packages/indexer-nibi/src/query/positions.ts

```diff
@@ -61,7 +61,7 @@ export const positions = async (
   }: GqlInPosition): string => {
     const argWhere = (): string => {
       const whereConditions: string[] = []
-      whereConditions.push(`pairEq: "${pair}"`)
+      if (trader) whereConditions.push(`pairEq: "${pair}"`)
```

Comment:
Is this supposed to say `      if (pair) whereConditions.push(`pairEq: "${pair}"`)`?

Reply:
yes, my mistake, just fixed that